doc/classes/TextureButton.xml

```diff
@@ -29,18 +29,21 @@
 		</member>
 		<member name="texture_disabled" type="Texture2D" setter="set_texture_disabled" getter="get_texture_disabled">
 			Texture to display when the node is disabled. See [member BaseButton.disabled].
+			If not assigned, when the [TextureButton] is disabled it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
 		</member>
 		<member name="texture_focused" type="Texture2D" setter="set_texture_focused" getter="get_texture_focused">
-			Texture to display when the node has mouse or keyboard focus. [member texture_focused] is displayed [i]over[/i] the base texture, so a partially transparent texture should be used to ensure the base texture remains visible. A texture that represents an outline or an underline works well for this purpose. To disable the focus visual effect, assign a fully transparent texture of any size. Note that disabling the focus visual effect will harm keyboard/controller navigation usability, so this is not recommended for accessibility reasons.
+			Texture to [i]overlay on the base texture[/i] when the node has mouse or keyboard focus. Because [member texture_focused] is displayed on top of the base texture, a partially transparent texture should be used to ensure the base texture remains visible. A texture that represents an outline or an underline works well for this purpose. To disable the focus visual effect, assign a fully transparent texture of any size. Note that disabling the focus visual effect will harm keyboard/controller navigation usability, so this is not recommended for accessibility reasons.
 		</member>
 		<member name="texture_hover" type="Texture2D" setter="set_texture_hover" getter="get_texture_hover">
 			Texture to display when the mouse hovers the node.
+			If not assigned, when the [TextureButton] is hovered over it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
 		</member>
 		<member name="texture_normal" type="Texture2D" setter="set_texture_normal" getter="get_texture_normal">
 			Texture to display by default, when the node is [b]not[/b] in the disabled, hover or pressed state. This texture is still displayed in the focused state, with [member texture_focused] drawn on top.
 		</member>
 		<member name="texture_pressed" type="Texture2D" setter="set_texture_pressed" getter="get_texture_pressed">
 			Texture to display on mouse down over the node, if the node has keyboard focus and the player presses the Enter key or if the player presses the [member BaseButton.shortcut] key.
+			If not assigned, when the [TextureButton] is pressed it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
```

Comment:
```suggestion
			Texture to display on mouse down over the node, if the node has keyboard focus and the player presses the Enter key or if the player presses the [member BaseButton.shortcut] key. If not assigned, displays [member texture_normal] instead.
```

Reply:
```suggestion
			Texture to display on mouse down over the node, if the node has keyboard focus and the player presses the Enter key or if the player presses the [member BaseButton.shortcut] key. If not assigned, displays [member texture_hover] instead.
```

Reply:
When `texture_pressed` and `texture_normal` are both assigned, and the TextureButton is pressed, then `texture_pressed` is used:

https://github.com/user-attachments/assets/e8a0e9ef-c181-4f14-a751-65c8bd1c03f0

When `texture_pressed` is not defined, but `texture_hover` and `texture_normal` are, and the TextureButton is pressed, **then `texture_hover` is used (*not* `texture_normal`)**:

https://github.com/user-attachments/assets/4d57983c-2f4e-4d30-8bbd-d118fd9c1c19

When neither `texture_pressed` nor `texture_normal` are assigned, and the TextureButton is pressed, then `texture_hover` is used:

https://github.com/user-attachments/assets/e3c90006-b331-4aaa-8a78-114e7bf0692d

Essentially, it looks like `texture_pressed` falls back to `texture_hover`, and that can in turn fall back to `texture_normal`.